packages/venia-concept/package.json

```diff
@@ -14,23 +14,18 @@
   },
   "homepage": "https://github.com/magento-research/pwa-studio/tree/master/packages/venia-concept#readme",
   "scripts": {
-    "build": "npm run -s validate-queries && webpack --color --progress --profile --env.phase production",
+    "build": "rimraf dist && npm run -s validate-queries && webpack --color --progress --profile --env.mode production",
```

Comment:
Does `&&` work on Windows? My cross-platform npm is a bit rusty.

Reply:
It does.